web/public/mgmt/config.yml

```diff
@@ -2997,10 +2997,13 @@ collections:
       - { label: "Name/Title", name: "name", widget: "string" }
       - label: "Icon"
         name: icon
-        widget: "select"
+        widget: "relation"
         multiple: false
```

Comment:
dust - I think the default is false, so you could remove `multiple: false`

Reply:
Good callout